docs/TechnicalIntroduction.rst

```diff
@@ -764,14 +764,14 @@ Supported compression schemes:
        input to the compressor is short, adding the header tends to offset any
        size reduction of the input.) 
    * - ZIPS (lossless)
-     - Uses the open source zlib library for compression. Unlike ZIP
+     - Uses the open source deflate library for zlib compression. Unlike ZIP
```

Comment:
we shouldn't say zlib here. zlib is just another implementation.

I believe the compression is IETF RFC 1952, which is normatively described, so let's go with that?

Reply:
sounds good, will fix up later

Reply:
was actually RFC 1950 - 1952 is gzip, not zlib (reference updated)